server.js

```diff
@@ -820,6 +825,25 @@ function determineWinner(thisUser){
     }
 }
 
+/**
+ * 入力値をチェックする
+ * @param {Array} rollList - 入力された役職リスト
+ * @param {Number} numPeople - プレイ人数
+ * @returns {Boolean}
+ */
+function isValidParameters(rollList, numPeople) {
+    // 役職リストチェック
+    for (var i = 0; i < rollList.length; i++) {
+        // 入力値が役職の範囲外なら正しくない
+        if (!rolls[rollList[i]]) return false;
+    }
+    // 人数チェック
+    // 二人未満なら正しくない
+    if (numPeople < 2) return false;
```

Comment:
```suggestion
    if (numPeople < 3) return false;
```
ワンナイト人狼の最少プレイ人数は3人なので3人未満かどうかを判断する。